### PR TITLE
download page for ipavo

### PR DESCRIPTION
**Pull request by tukwila**
ipavo offline pkg download page

### CODE REVIEW

**Review by windsonsea — COMMENTED** [review]
这2个地方也要增加 ipavo：
- download/index.md
- navigation.yml

File: docs/en/docs/download/modules/ipavo.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+# Ipavo
+
+On this page, you can download the offline packages of each version of the Ipavo module.
+
+## Download
+
+| Version  | Architecture | File Size | Package     | Checksum File      | Update Date |
+|----------|--------------|-----------|-------------|--------------------|-------------|
+| [v0.12.4] | AMD 64 | 51 MB | [:arrow_down: ipavo_v0.12.4_amd64.tar](https://qiniu-download-public.daocloud.io/DaoCloud_Enterprise/ipavo_v0.12.4_amd64.tar) | [:arrow_down: ipavo_v0.12.4_amd64_checksum.sha512sum](https://qiniu-download-public.daocloud.io/DaoCloud_Enterprise/ipavo_v0.12.4_amd64_checksum.sha512sum) | 2024-10-12 |
```
Comment (windsonsea) [inline docs/en/docs/download/modules/ipavo.md:16]:
```suggestion
| v0.12.4 | AMD 64 | 51 MB | [:arrow_down: ipavo_v0.12.4_amd64.tar](https://qiniu-download-public.daocloud.io/DaoCloud_Enterprise/ipavo_v0.12.4_amd64.tar) | [:arrow_down: ipavo_v0.12.4_amd64_checksum.sha512sum](https://qiniu-download-public.daocloud.io/DaoCloud_Enterprise/ipavo_v0.12.4_amd64_checksum.sha512sum) | 2024-10-12 |
```

File: docs/en/docs/download/modules/ipavo.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+| [v0.12.4] | ARM 64 | 51 MB | [:arrow_down: ipavo_v0.12.4_arm64.tar](https://qiniu-download-public.daocloud.io/DaoCloud_Enterprise/ipavo_v0.12.4_arm64.tar) | [:arrow_down: ipavo_v0.12.4_arm64_checksum.sha512sum](https://qiniu-download-public.daocloud.io/DaoCloud_Enterprise/ipavo_v0.12.4_arm64_checksum.sha512sum) | 2024-10-12 |
```
Comment (windsonsea) [inline docs/en/docs/download/modules/ipavo.md:17]:
```suggestion
| v0.12.4 | <font color="green">ARM 64</font>  | 51 MB | [:arrow_down: ipavo_v0.12.4_arm64.tar](https://qiniu-download-public.daocloud.io/DaoCloud_Enterprise/ipavo_v0.12.4_arm64.tar) | [:arrow_down: ipavo_v0.12.4_arm64_checksum.sha512sum](https://qiniu-download-public.daocloud.io/DaoCloud_Enterprise/ipavo_v0.12.4_arm64_checksum.sha512sum) | 2024-10-12 |
```

File: docs/zh/docs/download/modules/ipavo.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+# 仪表盘 Ipavo
+
+本页可下载仪表盘模块各版本的离线安装包。
+
+## 下载
+
+| 版本 | 架构 | 文件大小 | 安装包 | 校验文件 | 更新日期  |
+| ---- | --- | ------ | ----- | ------- | -------- |
+| [v0.12.4] | AMD 64 | 51 MB | [:arrow_down: ipavo_v0.12.4_amd64.tar](https://qiniu-download-public.daocloud.io/DaoCloud_Enterprise/ipavo_v0.12.4_amd64.tar) | [:arrow_down: ipavo_v0.12.4_amd64_checksum.sha512sum](https://qiniu-download-public.daocloud.io/DaoCloud_Enterprise/ipavo_v0.12.4_amd64_checksum.sha512sum) | 2024-10-12 |
```
Comment (windsonsea) [inline docs/zh/docs/download/modules/ipavo.md:14]:
```suggestion
| v0.12.4 | AMD 64 | 51 MB | [:arrow_down: ipavo_v0.12.4_amd64.tar](https://qiniu-download-public.daocloud.io/DaoCloud_Enterprise/ipavo_v0.12.4_amd64.tar) | [:arrow_down: ipavo_v0.12.4_amd64_checksum.sha512sum](https://qiniu-download-public.daocloud.io/DaoCloud_Enterprise/ipavo_v0.12.4_amd64_checksum.sha512sum) | 2024-10-12 |
```

File: docs/zh/docs/download/modules/ipavo.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+| [v0.12.4] | ARM 64 | 51 MB | [:arrow_down: ipavo_v0.12.4_arm64.tar](https://qiniu-download-public.daocloud.io/DaoCloud_Enterprise/ipavo_v0.12.4_arm64.tar) | [:arrow_down: ipavo_v0.12.4_arm64_checksum.sha512sum](https://qiniu-download-public.daocloud.io/DaoCloud_Enterprise/ipavo_v0.12.4_arm64_checksum.sha512sum) | 2024-10-12 |
```
Comment (windsonsea) [inline docs/zh/docs/download/modules/ipavo.md:15]:
```suggestion
| v0.12.4 | <font color="green">ARM 64</font>  | 51 MB | [:arrow_down: ipavo_v0.12.4_arm64.tar](https://qiniu-download-public.daocloud.io/DaoCloud_Enterprise/ipavo_v0.12.4_arm64.tar) | [:arrow_down: ipavo_v0.12.4_arm64_checksum.sha512sum](https://qiniu-download-public.daocloud.io/DaoCloud_Enterprise/ipavo_v0.12.4_arm64_checksum.sha512sum) | 2024-10-12 |
```

File: docs/en/docs/download/modules/ipavo.md (hunk 1 above)
Comment (tukwila) [reply docs/en/docs/download/modules/ipavo.md]: fix

File: docs/en/docs/download/modules/ipavo.md (hunk 2 above)
Comment (tukwila) [reply docs/en/docs/download/modules/ipavo.md]: fix

File: docs/zh/docs/download/modules/ipavo.md (hunk 3 above)
Comment (tukwila) [reply docs/zh/docs/download/modules/ipavo.md]: fix

File: docs/zh/docs/download/modules/ipavo.md (hunk 4 above)
Comment (tukwila) [reply docs/zh/docs/download/modules/ipavo.md]: fix

**Review by windsonsea — APPROVED** [review]

**Pull request merged**